lib/AST/CMakeLists.txt

```diff
@@ -73,6 +77,7 @@ add_swift_library(swiftAST STATIC
     clangLex
     clangAPINotes
     clangBasic
+    ${mincore}
```

Comment:
We have target specific link dependencies, you should use that.

Im confused why this is needed in swift but not in clang.